src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --colorgrey: #e1e7ed;
+  --dropcolor: #00000040;
+  --fontcolor: #3d4e61;
+}
+
+.search {
+  margin-top: 20px;
+}
+
+.search__input {
+  width: 100%;
+  box-sizing: border-box;
+  background-image: url(/src/images/Search.svg);
+  background-repeat: no-repeat;
+  color: var(--fontcolor);
+  border: 1px solid var(--colorgrey);
+  border-radius: 4px;
+  outline: none;
+  box-shadow: 0 1px 8px #3d4e611a;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search__input::placeholder {
+  color: var(--fontcolor);
+}
+
+.search__input--big {
+  height: 70px;
+  padding: 0 62px;
+  background-size: 19px 19px;
+  background-position: 26px center;
+  font-size: 16px;
+}
+
+.search__input--small {
+  height: 42px;
+  padding: 0 33px;
+  background-position: 13px center;
+  background-size: 11px 11px;
+  font-size: 14px;
+}
+
+.search__input--small:hover,
+.search__input--big:hover {
+  box-shadow: 0 3px 8px 0 var(--dropcolor);
+  font-family: Avenir-Book, sans-serif;
+}
```

Comment:
If both modifiers apply the same style you should consider moving them to the block/element instead